scipy/spatial/__init__.py

```diff
@@ -97,6 +106,7 @@
 from ._spherical_voronoi import SphericalVoronoi
 from ._plotutils import *
 from ._procrustes import procrustes
+from .transform import *
```

Comment:
If I'm not mistaken we want to use `spatial.transform` namespace? In this case looks like you need to do `from . import transform`.

Reply:
Oh, I didn't think of that.